tla/consensus/MCccfraft.tla

```diff
@@ -129,17 +129,8 @@ DebugNotTooManySigsInv ==
 ----
 \* Refinement
 
-\* The inital log is up to 4 entries long + one log entry per request/reconfiguration + one signature per request/reconfiguration or new view (no consecutive sigs except across views)
-MaxLogLength == 
-    4 + ((RequestLimit + Len(Configurations)) * 2) + MaxTermLimit
-
-MappingToAbs == 
-  INSTANCE abs WITH
-    Servers <- Servers,
-    Terms <- StartTerm..MaxTermLimit,
-    MaxLogLength <- MaxLogLength,
-    cLogs <- [i \in Servers |-> [j \in 1..commitIndex[i] |-> log[i][j].term]]
-
-RefinementToAbsProp == MappingToAbs!AbsSpec
+\* A follower can get RequestLimit entries behind.  However, each request may be followed by a
+\* a signature. Thus, we need to allow for 2*RequestLimit entries.
+MCRefinementToAbsProp == MappingToAbs(StartTerm..MaxTermLimit, 2 * RequestLimit)!AbsSpec
```

Comment:
Why is `MaxUncommittedCount` limited to `2*RequestLimit`? I think it could be more e.g. when a leader adds reconfiguration transaction

Reply:
Surprisingly, I tested `MCccraftWithReconfig` locally and it passed.

Reply:
Yep, I'm seeing the same locally. It's passing `MCccraftWithReconfig` even when I set `RequestLimit = 0` and thus disabling `Extend`

Reply:
With https://github.com/microsoft/CCF/pull/6475/commits/5ec9496e3e4d80f4ab6886f7bb8e22d158ae1e3f it fails.

Reply:
I see, it was only passing as I forgot to add `RefinementToAbsProp` to properties